src/sass/components/ConfirmationModal.scss

```diff
@@ -0,0 +1,35 @@
+.confirmation-modal, .confirmation-blurred-blocker {
+  position: absolute;
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+}
+
+.confirmation-modal {
+  width: 30%;
+  min-width: 320px;
+  height: 200px;
+  margin: auto;
+  position: fixed;
+  z-index:3;
+  background:#e6e7e9;
+  border-radius: 4px;
+  box-shadow: 0 0 4px -1px #525652, 0 1px 2px -1px #525652;
+}
+
+.confirmation-modal > div {
+  width: 145px;
+  margin: 50px auto 0;
+}
+
+.confirmation-blurred-blocker {
+  background: rgba(255,255,255,0);
```

Comment:
This is picky, but the base color for black is `rgb(33,33,33)` in USWDS if you want to use that instead.